lib/src/specs/flex/flex_util.dart

```diff
@@ -78,22 +78,22 @@ class FlexSpecUtility<T extends SpecAttribute>
   /// Returns a `TextBaselineUtility` to set the alignment.
   TextBaselineUtility<T> get textBaseline {
     return TextBaselineUtility(
-      (textBaseline) => only(textBaseline: textBaseline),
+      (textBaseline) => call(textBaseline: textBaseline),
     );
   }
 
   /// Sets the clip behavior.
   ///
   /// Returns a `ClipUtility` to set the behavior.
   ClipUtility<T> get clipBehavior {
-    return ClipUtility((clipBehavior) => only(clipBehavior: clipBehavior));
+    return ClipUtility((clipBehavior) => call(clipBehavior: clipBehavior));
```

Comment:
This goes against the convention of Dtos and the Flutter classes. Can you revert this, and I can merge the other changes.